deep_quoridor/src/quoridor_env.py

```diff
@@ -547,3 +592,38 @@ def env(**kwargs):
         return wrappers.CaptureStdoutWrapper(QuoridorEnv(**kwargs))
     else:
         return QuoridorEnv(**kwargs)
+
+
+class StepRewardCalculator:
```

Comment:
I like having the step rewards as a separate class! Should make it easier to have various different reward functions in the future.